tests/50federation/31room-send.pl

```diff
@@ -145,3 +146,71 @@
          });
       });
    };
+
+test "Ephemeral messages received from servers are correctly expired",
+   requires => [ local_user_and_room_fixtures(), federation_user_id_fixture(),
+                 $main::OUTBOUND_CLIENT ],
```

Comment:
Note that sytest spins up *two* synapse instances, and where we're not doing anything magical with the protocol that makes it hard to do so, generally I prefer to test federation stuff by sending messages between those two servers rather than using the sytest federation impl: it gives a better end-to-end test. You can create users on the second server with `remote_user_fixture`: see other tests for further details.

Again, a thought for the future rather than one that needs changing here.